### PR TITLE
Get anchor links to work

### PR DESCRIPTION
**Pull request by mikelmaron**
@brunosan, I can squash this later into one commit, if you prefer. in the mean time, take a look, and see if this looks ok. http://mikelmaron.github.io/web/list/?data=full_list#goal-3-a

### CODE REVIEW

File: _sass/_layout.scss [hunk 1]
```diff
@@ -234,3 +234,10 @@
         }
     }
 }
+
+a.anchor {
+    display: block;
+    position: relative;
+    top: -75px;
+    visibility: hidden;
+}
```
Comment (brunosan) [inline _sass/_layout.scss:243]: This is the magic :)

File: _config.yml [hunk 2]
```diff
@@ -5,6 +5,6 @@ description: > # this means to ignore newlines until "baseurl:"
   This website aggregates and offers the data in raw format, but also
   visualized and open to contributions.
 baseurl: "/web" # the subpath of your site, e.g. /blog/
-github_username:  SDG-data
+github_username: mikelmaron
```
Comment (brunosan) [inline _config.yml]: this needs to be changed for the merge

**Comment by brunosan** [comment]
Awesome! Thanks!
Extra kudos for contributing to this spaghetti code. I think probably Jekyll is an overkill here, and there's too much js DOMing, but I guess it works for now so...

I'll put you in team admin so you can merge after changing the config github username.

cc @pythagoraswitch

**Comment by mikelmaron** [comment]
ok I've changed the github username. the tricky part was preserving and then acting on the anchor after the DOM was built.

**Comment by mikelmaron** [comment]
oh @brunosan, still don't have write access...

**Comment by pythagoraswitch** [comment]
Thank you so much, @mikelmaron  - FYI you should have write access via membership to @SDG-data/collaborators. Invitation was sent to you via @brunosan. Let me know if it still doesn't work.

**Comment by pythagoraswitch** [comment]
@mikelmaron Thank you so much for your magical code! It looks awesome!

**Pull request merged**